### DISCUSSION TITLE
How does the performance compare to go-sqlite with CGO?

### DISCUSSION THREAD

**Discussion by SakethAjith** [ncruces/go-sqlite3#155] [Q&A] [answered]
Hello!, I love CGO less deps,
I found [this](https://www.reddit.com/r/golang/comments/11wmshw/comment/je0ksls/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) reddit comment you made 2 years ago.

* Have the mentioned optimizations on **wazero** and **wasm** been made?
* How does this compare now to `mattn/go-sqlite3` in regards to performance?

**Comment by ncruces** [comment, marked as answer]
There's a link in the README to this benchmark:
https://github.com/cvilsmeier/go-sqlite-bench

As for wazero, yes they [were](https://github.com/tetratelabs/wazero/releases/tag/v1.7.0). I am uninterested in Wasm threads, the current instance per connection works fine.
